core/common/src/main/java/alluxio/LeaderInquireClient.java

```diff
@@ -106,7 +106,8 @@ public synchronized String getMasterAddress() {
         CommonUtils.sleepMs(LOG, Constants.SECOND_MS);
       }
     } catch (Exception e) {
-      LOG.error("Error with zookeeper: {} message: {}", mZookeeperAddress, e.getMessage());
+      LOG.error("Error getting the master address from zookeeper. Zookeeper address: {}",
+          mZookeeperAddress, e.getMessage());
```

Comment:
Does `e.getMessage()` end up in the error message anywhere? Is there a `{}` missing?